src/algebra/pointwise.lean

```diff
@@ -437,4 +437,13 @@ lemma mul_subset_closure {s t u : set M} (hs : s ⊆ u) (ht : t ⊆ u) :
   s * t ⊆ submonoid.closure u :=
 mul_subset (subset.trans hs submonoid.subset_closure) (subset.trans ht submonoid.subset_closure)
 
+@[to_additive]
+lemma submonoid_mul_self_eq (s : submonoid M) : (s : set M) * s = s :=
```

Comment:
```suggestion
lemma coe_mul_self_eq (s : submonoid M) : (s : set M) * s = s :=
```

Else this lemma is called `submonoid.submonoid_mul_self_eq` which is repetitive.

Does it make sense to also include this lemma in `group_theory/submonoid/operations.lean`, without the coercion? Or do submonoids not have `has_mul` (as an alias for "closure of product of set coercions", like submodule has)?

Reply:
Did you find such an instance?
```
example (M : Type*) [monoid M] : has_mul (submonoid M) := by apply_instance
```
turns up nothing for me.

Reply:
No, I concluded the instance doesn't exist.

Reply:
Marking as unresolved, the comment about naming still applies.

Reply:
Oops, thanks!